### ISSUE TITLE
Setting output size?

### ISSUE THREAD

**Issue by syntaqx** [yeqown/go-qrcode#107]
I realize I'm able to set the `QRWidth` using:
```
standard.WithQRWidth(255)
```

However, how can I specify that I want a total image output of say, `255x255`?

**Comment by yeqown** [comment]
If I understand you correctly，`1024x1024` means the output image pixels size which isn't able to be limited since your input content would be oversized, so we don't provide such an option yet.  @syntaqx

**Comment by syntaqx** [comment]
Yes, I'm referring to the output image's pixel size. I want to use your library, as I was previously using https://github.com/skip2/go-qrcode, which allowed me to simply do:

```
var png []byte
png, err := qrcode.Encode("https://example.org", qrcode.Medium, 1024)
```

Any plans to make this feature?
